### PR TITLE
[jest-gas-reporter] Lint jest-gas-reporter

### PR DESCRIPTION
**Pull request by snario**
Adds a super basic eslint config and fixes linting issues with it.

### CODE REVIEW

File: packages/jest-gas-reporter/.eslintrc.js [hunk 1]
```diff
@@ -1,178 +1,16 @@
...
-    'plugin:import/warnings',
-    'plugin:import/typescript',
-    'prettier/@typescript-eslint',
-    'prettier',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended'
   ],
-  rules: Object.assign(rules, TSRules),
+  rules: {
+    'prettier/prettier': ['warn']
```
Comment (Alonski) [inline packages/jest-gas-reporter/.eslintrc.js]: Hmm why not error?

File: packages/jest-gas-reporter/package.json [hunk 2]
```diff
@@ -12,8 +12,8 @@
     "prepare": "yarn build",
     "build": "yarn lint:check && tsc -b",
     "build:typescript": "tsc -b",
-    "lint:check": "eslint \"*/**/*.ts\" --cache",
-    "lint:write": "eslint \"*/**/*.ts\" --fix"
+    "lint:check": "eslint \"src/**/*.ts\" --cache",
```
Comment (Alonski) [inline packages/jest-gas-reporter/package.json]: Why is this only checking `src` and `ts` files?
Do we not want linting for all files?

File: packages/jest-gas-reporter/.eslintrc.js (hunk 1 above)
Comment (snario) [reply packages/jest-gas-reporter/.eslintrc.js]: To distinguish between linting issues and build issues. I think it makes the code more readable.

Comment (Alonski) [reply packages/jest-gas-reporter/.eslintrc.js]: The question is do we want CI to fail on formatting issues?
Anyways your choice :)

Comment (snario) [reply packages/jest-gas-reporter/.eslintrc.js]: Good point. Turns out we can have the best of both worlds with `--max-warnings=0`.

Screen Shot 2019-12-06 at 10 50 08
Screen Shot 2019-12-06 at 10 50 25

**Pull request merged**